src/core/DataHelper.ttslua

```diff
@@ -550,28 +546,24 @@ function updateHiddenCards(args)
 end
 
 --------------------------------------------------------------------------
--- deprecated code (commented out)
+-- deprecated code
 --------------------------------------------------------------------------
 
 -- deprecated, use metadata (GM Notes) instead
 -- Known locations and clues. We check this to determine if we should atttempt to spawn clues,
 -- first we look for <LOCATION_NAME>_<GUID> and if we find nothing we look for <LOCATION_NAME>
 -- format is [location_guid -> clueCount]
--- LOCATIONS_DATA = JSON.decode([[{
---   "XXXX": {"type": "fixed", "value": 2, "clueSide": "back"},
---   "xxx": {"type": "perPlayer", "value": 2, "clueSide": "back"}
--- }]])
+LOCATIONS_DATA = JSON.decode([[{
+  "XXXX": {"type": "fixed", "value": 2, "clueSide": "back"},
```

Comment:
This is because the table is referenced and is nil if unassigned, right?  Let's remove these dummy values from the actual code, and beef up the documentation to include specific examples.  Same for below.

Reply:
Updated